### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by ahankinson**

### CODE REVIEW

File: default/de/publication_520.md [hunk 1]
```diff
@@ -10,5 +10,5 @@ Enter the beginning date and (if applicable) the ending date of an item. If the
 
 ##### Examples
 
-- Vol. 1, no. 1 (Apr. 1981)-
+- Vol. Vol. 1, no. 1 (Apr. 1981)-
```
Comment (ahankinson) [inline default/de/publication_520.md:13]: @lpugin There's something weird going on here -- a lot of things are duplicated (see the "Ph.D." above too... )

Comment (lpugin) [reply default/de/publication_520.md:13]: Hmm... I haven't change anything in there. At least I did not mean to....

Comment (ahankinson) [reply default/de/publication_520.md:13]: It seems like there's something messed up in Crowdin?

Comment (ahankinson) [reply default/de/publication_520.md:13]: image

**Comment by ahankinson** [comment]
@lpugin I think this branch needs attention -- I'm not sure if the changes that will be introduced here are desired or not.

**Pull request merged**